code/game/objects/items/ego_weapons/he.dm

```diff
@@ -589,6 +589,69 @@
 	if(force < 100)
 		force += 0.25 //It charges pretty slowly, but people walk pretty fast thanks to justice.
 
+/obj/item/ego_weapon/lance/brick_road
+	name = "brick road"
+	desc = "To the wizard we go!"
+	special = "This weapon leaves slowing yellow bricks from charging"
+	icon_state = "brick_road"
+	lefthand_file = 'icons/mob/inhands/96x96_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/96x96_righthand.dmi'
+	damtype = WHITE_DAMAGE
+	armortype = WHITE_DAMAGE
+	force = 50
+	inhand_x_dimension = 96
+	inhand_y_dimension = 96
+	attack_speed = 4//Really really slow.
+	attack_verb_continuous = list("stabs", "impales")
+	attack_verb_simple = list("stab", "impale")
+	hitsound = 'sound/weapons/ego/spear1.ogg'
+	attribute_requirements = list(
+							JUSTICE_ATTRIBUTE = 40
+							)
+	reach = 2
+	force_cap = 100 //Old max damage when it was damage = amount you walked.
+	force_per_tile = 5 //if I can read, this means you need to cross 20 tiles for max damage
+	pierce_force_cost = 20
+	charge_speed_cap = 2//more of a jog then a ram
+	couch_cooldown_time = 5 SECONDS
+
+/obj/item/ego_weapon/lance/brick_road/UserMoved(mob/user)
+	..()
+	if(!raised)
+		playsound(src, 'sound/abnormalities/roadhome/House_MakeRoad.ogg', 100, FALSE, 8)
+		new /obj/effect/golden_road2(get_turf(user))
+
+//Not an actual floor, but an effect you put on top of it.
+/obj/effect/golden_road2
+	name = "Golden Road"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "gold" //note : find a proper brick road sprite later
+	alpha = 0
+	anchored = TRUE
+	var/slow_down = 0.75//25% slow
+	var/list/faction = list("neutral")
+
+/obj/effect/golden_road2/Initialize()
+	. = ..()
+	QDEL_IN(src, 30 SECONDS)
+	animate(src, alpha = 255,transform= transform, time = 0.5 SECONDS)
+	addtimer(CALLBACK(src, .proc/FadeOut), 29.5 SECONDS)
+	for(var/i = 1 to 30)
+		addtimer(CALLBACK(src, .proc/Slow), i * 1 SECONDS)
```

Comment:
```suggestion
	addtimer(CALLBACK(src, .proc/Slow), 1 SECONDS, TIMER_LOOP)
```
TIMER_LOOP should be used when something wants to run a proc until it qdel's. Note that this can be combined with TIMER_STOPPABLE and an assigned variable `var a = addtimer(...)` to make a looping conditional with the proc. Old coders warn not to abuse this